massgov/mayflower
# DP-22559 Fix hamburger menu errors in Safari

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description
As part of a previous ticket, the google translate element is moved depending on the browser size, there is a piece of js that is changing the tabindex of the main nav translate element when on mobile that it's not longer relevant. 
See: https://jira.mass.gov/browse/DP-22281

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-22559)

## Steps to Test


1. Open the homepage on Safari.
2. Select Develop > Enter Responsive Design Mode in the Safari toolbar.
3. On a viewport smaller than 840px , click the menu.
4. Check the console (right click > inspector > click "console" tab) 

There shouldn't be javascript errors.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* Header.

#### @TODO


*

#### Today I learned...
